xbmc/cores/VideoPlayer/VideoPlayer.cpp

```diff
@@ -4629,6 +4641,13 @@ void CVideoPlayer::GetVideoStreamInfo(int streamId, SPlayerVideoStreamInfo &info
   if (s.name.length() > 0)
     info.name = s.name;
 
+  if (s.width == 0 || s.height == 0)
```

Comment:
According to @fritsch checking width and height against zero is no good idea to check validity. He told me that tvheadend "sometimes" returns zero dimensions for "certain" channels. Maybe to workaround this tvh bug checking against string length of info.videoCodecName is acceptable?